modules/manage/pages/manage-xdcr/replicate-using-scopes-and-collections.adoc

```diff
@@ -0,0 +1,178 @@
+
+As well as specifying a source bucket and a target bucket, XDCR can specify scopes and collections within those buckets.
+This allows data to be reassigned from one collection to another.
+It also provides a convenient way of _migrating_ data that currently resides in legacy buckets that do not support (or have not yet been established with) administrator-defined scopes and collections, into buckets, on a Version 7.0+ Couchbase-Server cluster, that _do_ contain administrator-defined scopes and collections.
+For more information, see xref:learn:clusters-and-availability/xdcr-overview.adoc#xdcr-with-scopes-and-collections[XDCR with Scopes and Collections].
+
+[#examples-on-this-page]
+== Examples on this Page
+The examples provided on this page &#8212; xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#migrate-data-to-a-collection[Migrate Data to a Collection] and xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#replicate-data-between-collections[Replicate Data Between Collections] &#8212; demonstrate how XDCR can be performed with reference to scopes and collections within source and target buckets, using Couchbase Web Console.
+The examples assume the following:
+
+* A single-node cluster named `10.144.210.101` has been created, and features the `travel-sample` bucket.
+The `travel-sample` bucket contains, by default, only a `_default` scope and a `_default` collection; within which all its data resides.
+See xref:manage:manage-settings/install-sample-buckets.adoc[Install Sample Buckets], for information on installing `travel-sample`.
+
+* A single-node cluster named `10.144.210.102` has been created, and features a single, empty bucket, named `ts`.
+This bucket contains a scope named `airlineScope`; which itself contains a collection named `airlineCollection`.
+For information on creating scopes and collections, see xref:manage:manage-scopes-and-collections/manage-scopes-and-collections.adoc[Manage Scopes and Collections].
+
+* A single-node cluster named `10.144.210.103` has been created, and features a single, empty bucket, named `tsBackup`.
+This bucket contains a scope named `usAirlineBackupScope`; which itself contains a collection named `usAirlineBackupCollection`.
+
+* The cluster `10.144.210.101` has defined `10.144.210.102` as a remote cluster for XDCR.
+In turn, the cluster `10.144.210.102` has defined `10.144.210.103` as a remote cluster for XDCR.
+
+Before following the examples, familiarity should be gained with XDCR management-basics, explained on other pages. In particular, see xref:manage:manage-xdcr/prepare-for-xdcr.adoc[Prepare for XDCR], xref:manage:manage-xdcr/create-xdcr-reference.adoc[Create a Reference], xref:manage:manage-xdcr/create-xdcr-replication.adoc[Create a Replications], and xref:manage:manage-xdcr/filter-xdcr-replication.adoc[Filter a Replication].
+
+[#migrate-data-to-a-collection]
+== Migrate Data to a Collection
```

Comment:
I understand. Unfortunately, the problem here is that at this moment, I don't have a starting point for the examples without migration from _default. Because the reader is likely to have no collections yet set up - or at least, has not yet populated them. The plan is to have a new version of travel_sample that will be collections ready: so it would make sense to use that, and go directly to the implicit/explicit example, with source collections ready to go. But at this moment, the new sample bucket is not yet available.  When it arrives, I can make the change. (Or is there another way around this, I wonder?)